transaction_integration_test.go

```diff
@@ -241,6 +241,10 @@ func TestTransactionCreateWhenGatewayRejectedFraud(t *testing.T) {
 		t.Fatalf("Got status %q, want %q", txn.Status, "gateway_rejected")
 	}
 
+	if txn.GatewayRejectionReason != GatewayRejectionReasonFraud {
+		t.Fatalf("Got gateway rejection reason '%s', wanted 'fraud'", txn.GatewayRejectionReason)
```

Comment:
If you replace `'%s'` with `%q` you'll get a quoted string out the other side, which is consistent with the other expectations in this test file. e.g. https://play.golang.org/p/bCG-ipotJv